tests/test_wps_convert.py

```diff
@@ -38,11 +46,42 @@ def test_wps_convolution_local(uhs_files, station_file, domain, config_file):
         )
     ],
 )
-def test_wps_convolution_https(uhs_files, station_file, domain, config_file):
-    params = (
-        f"uhs_files=@xlink:href={uhs_files};"
-        f"station_file=@xlink:href={station_file};"
-        f"domain=@xlink:href={domain};"
-        f"config_file=@xlink:href={config_file};"
-    )
+def test_wps_convert_https(uhs_files, station_file, domain, config_file):
+    params = build_params(uhs_files, station_file, domain, config_file)
     run_wps_process(Convert(), params)
+
+
+@mark.online
+@mark.parametrize(
+    ("uhs_files", "station_file", "domain", "config_file"),
+    [
+        (
+            local_path("samples/sample.uh_s2"),
+            local_path("samples/station_file.txt"),
+            url_path("sample_routing_domain.nc", "http", "climate_explorer_data_prep"),
+            # convolve.cfg has wrong keys
+            local_path("configs/convolve.cfg"),
+        )
+    ],
+)
+def test_wps_convert_config_err(uhs_files, station_file, domain, config_file):
+    params = build_params(uhs_files, station_file, domain, config_file)
+    process_err_test(Convert(), params)
+
+
+@mark.online
+@mark.parametrize(
+    ("uhs_files", "station_file", "domain", "config_file"),
+    [
+        (
+            local_path("samples/sample.uh_s2"),
+            local_path("samples/station_file.txt"),
+            url_path("sample_routing_domain.nc", "http", "climate_explorer_data_prep"),
+            # convert[UHS_FILES][ROUT_PROGRAM] is Python (should be C pr Fortran)
```

Comment:
@nikola-rados is this comment okay because I also gave the function a descriptive name, or should I get rid of all comments like this in the parameters?

Reply:
once I fix the "pr" of course

Reply:
Generally the comments are _fine._ I was just pointing out a way that you could make the code more self-documenting.